src/commands/generate.rs

```diff
@@ -36,3 +37,10 @@ fn command(name: &str, binary_path: PathBuf, args: &[&str]) -> Command {
     c.args(args);
     c
 }
+
+fn project_type(template: &str) -> ProjectType {
+    if template.contains("rust") {
```

Comment:
This should probably be:

```suggestion
    if template == "rust" {
```

Reply:
no- this is matching the url so contains is correct here